loopout.go

```diff
@@ -705,8 +709,27 @@ func (s *loopOutSwap) payInvoiceAsync(ctx context.Context,
 			pluginType, hash.String())
 
 		maxRetries = s.executeConfig.maxPaymentRetries
-		paymentTimeout /= time.Duration(maxRetries)
+
+		// If not set, default to the per payment timeout to the total
+		// payment timeout divied by the configured maximum retries.
+		if paymentTimeout == 0 {
+			paymentTimeout = totalPaymentTimeout /
+				time.Duration(maxRetries)
+		}
+
+		// If the payment timeout is too long, we need to adjust the
+		// number of retries to ensure we don't exceed the total
+		// payment timeout.
+		if paymentTimeout*time.Duration(maxRetries) >
+			totalPaymentTimeout {
+
+			maxRetries = int(totalPaymentTimeout / paymentTimeout)
```

Comment:
If totalPaymentTimeout is not divisible by paymentTimeout, the remainder of totalPaymentTimeout will be removed. Does it make sense for the last attempt to use a shorter timeout instead?